servers/src/main/java/tachyon/master/TachyonMaster.java

```diff
@@ -181,6 +181,10 @@ protected TachyonMaster() {
       MasterContext.getMasterSource().registerGauges(this);
       mMasterMetricsSystem = new MetricsSystem("master", MasterContext.getConf());
       mMasterMetricsSystem.registerSource(MasterContext.getMasterSource());
+
+      mWebServer =
```

Comment:
I think it's good practice anyway to set fields like this in the constructor. I just added this comment here to make it clear why I was moving it in this PR specifically.